### ISSUE TITLE
Issues with plugin - incompatible types

### ISSUE THREAD

**Issue by archenroot** [dmak/jaxb-xew-plugin#56]
Hi,

I work on project using your plugin inside maven-jaxb2-plugin and facing some issue with incompatible types, not sure how to change XSD to fix this.

The whole project is available at:
http://ip-dev01.coreso.eu/gitlab/architecture/enterprise-integration/common-data-model

the pom.xml I am having issue with is under cdm-pojo module. The path to schemas is defined in the plugin configuration and points to cdm-schemas/.... directory(ies).

Thanks for hint and this plugin.

If you run mvn -X clean install you will hit the errors, but just for some small reference here is log snippet:
```
[ERROR] /home/zangetsu/proj/coreso/common-data-model/cdm-pojo/target/generated-sources/xjc/eu/coreso/integration/integrationobjectlibrary/core/common/v1/CommonMessageHeaderType.java:[328,47] incompatible types: java.util.List<eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertyType> cannot be converted to eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertiesType
[ERROR] /home/zangetsu/proj/coreso/common-data-model/cdm-pojo/target/generated-sources/xjc/eu/coreso/integration/integrationobjectlibrary/core/common/v1/CommonMessageHeaderType.java:[400,47] incompatible types: java.util.List<eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertyType> cannot be converted to eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertiesType
[ERROR] /home/zangetsu/proj/coreso/common-data-model/cdm-pojo/target/generated-sources/xjc/eu/coreso/integration/integrationobjectlibrary/core/common/v1/CommonMessageHeaderType.java:[402,47] incompatible types: java.util.List<eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertyType> cannot be converted to eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertiesType
[ERROR] /home/zangetsu/proj/coreso/common-data-model/cdm-pojo/target/generated-sources/xjc/eu/coreso/integration/integrationobjectlibrary/core/common/v1/CommonMessageHeaderType.java:[475,47] incompatible types: java.util.List<eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertyType> cannot be converted to eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertiesType
[ERROR] /home/zangetsu/proj/coreso/common-data-model/cdm-pojo/target/generated-sources/xjc/eu/coreso/integration/integrationobjectlibrary/core/common/v1/CommonMessageHeaderType.java:[575,58] incompatible types: java.util.List<eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertyType> cannot be converted to eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertiesType
[ERROR] /home/zangetsu/proj/coreso/common-data-model/cdm-pojo/target/generated-sources/xjc/eu/coreso/integration/integrationobjectlibrary/core/common/v1/CommonMessageHeaderType.java:[577,40] incompatible types: eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertiesType cannot be converted to java.util.List<eu.coreso.integration.integrationobjectlibrary.core.common.v1.PropertyType>
[ERROR] /home/zangetsu/proj/coreso/common-data-model/cdm-pojo/target/generated-sources/xjc/eu/coreso/integration/integrationobjectlibrary/core/common/v1/MessageIdentificationType.java:[218,13] cannot find symbol
  symbol:   class BusinessObjectDocuments
  location: class eu.coreso.integration.integrationobjectlibrary.core.common.v1.MessageIdentificationType
[ERROR] /home/zangetsu/proj/coreso/common-data-model/cdm-pojo/target/generated-sources/xjc/eu/coreso/integration/integrationobjectlibrary/core/common/v1/MessageIdentificationType.java:[270,13] cannot find symbol
  symbol:   class BusinessObjectDocuments

```

**Comment by dmak** [comment]
Sorry for looking into the issue after a considerable time since it was reported.

I am not able to open the link with the example. Could you please update it or, better, attach XSD directly to the bug issue? Thanks!

**Comment by dmak** [comment]
Closing as test case is missing. Probably was solved in v1.10 of the plugin.
